app/services/tock_importer.rb

```diff
@@ -6,6 +6,7 @@ def perform
       tock_project = ClientAccount.find_or_initialize_by(tock_id: project["id"])
       tock_project.name = project["name"]
       tock_project.billable = project["billable"]
+      tock_project.live = project["live"]
```

Comment:
I think that rather than `respond_to` I'd make sure that `ClientAccount` records are being set to `active` properly